packages/client-sdk/src/Mash.ts

```diff
@@ -66,6 +66,9 @@ class Mash {
             fontFamily: "inherit",
           });
         }
+        console.warn(
+          "[MASH] Error when fetching wallet placement from API, using default placement",
+        );
```

Comment:
super nit: can we bring this to be right at the top of the `catch` handler, and change the message to
```suggestion
        console.warn(
          "[MASH] Error when fetching wallet placement from API, using default placement and theme",
        );
```

Reply:
all good, just pushed